File: Python/solve.py
def operaciones_matriciales(n):
    # Crear matrices de prueba
    matriz_a = [[i + j for j in range(n)] for i in range(n)]
    matriz_b = [[j - i for j in range(n)] for i in range(n)]
    
    # Realizar operaciones matriciales básicas
    import time
    inicio = time.time()
    
    # Multiplicación de matrices
    resultado = [[0 for _ in range(n)] for _ in range(n)]
    for i in range(n):
        for j in range(n):
            for k in range(n):
                resultado[i][j] += matriz_a[i][k] * matriz_b[k][j]
    
    fin = time.time()
    tiempo_ejecucion = (fin - inicio) * 1000  # Convertir a milisegundos
    
    # Guardar el resultado en un archivo
    with open("resultadoPy.txt", "w") as archivo:
        archivo.write(f"Tiempo de ejecución para n={n}: {tiempo_ejecucion:.4f} ms\n")
    
    return tiempo_ejecucion

# Ejecutar la función y guardar el resultado
n = 100
operaciones_matriciales(n)

File: Javascript/solve.js
const fs = require('fs');  // Importa el módulo 'fs' para trabajar con archivos

function operacionesMatriciales(n) {
    // Crear matrices de prueba
    let matriz_a = Array(n).fill().map((_, i) => 
        Array(n).fill().map((_, j) => i + j)
    );
    
    let matriz_b = Array(n).fill().map((_, i) => 
        Array(n).fill().map((_, j) => j - i)
    );
    
    // Realizar operaciones matriciales básicas
    const inicio = performance.now();
    
    // Multiplicación de matrices
    let resultado = Array(n).fill().map(() => 
        Array(n).fill().map(() => 0)
    );
    
    for (let i = 0; i < n; i++) {
        for (let j = 0; j < n; j++) {
            for (let k = 0; k < n; k++) {
                resultado[i][j] += matriz_a[i][k] * matriz_b[k][j];
            }
        }
    }
    
    const fin = performance.now();
    return fin - inicio;
}

// Guardar el resultado en un archivo
const tiempo = operacionesMatriciales(100).toFixed(4);
fs.writeFileSync('resultadoJs.txt', `Javascript: ${tiempo} ms\n`);
